gpflow/base.py

```diff
@@ -307,6 +304,19 @@ def _run_op(a, *args, **kwargs):
 Parameter._OverloadAllOperators()
 tf.register_tensor_conversion_function(Parameter, lambda x, *args, **kwds: x.read_value())
 
+TensorLike = Union[np.ndarray, tf.Tensor, tf.Variable, Parameter]
+"""
+Type alias for tensor-like types that are supported by most TensorFlow, NumPy and GPflow operations.
+
+NOTE: Union types like this do not work with the `register` method of multipledispatch's
+`Dispatcher` class. Instead use `TensorLikeTypes` for dispatching on tensor-like types.
+"""
+
+# We've left this as object until we've tested the performance consequences of using the full set
+# see github issue #1434
```

Comment:
```suggestion
# see https://github.com/GPflow/GPflow/issues/1434
```
I think it's generally more helpful to have the full link there - one click instead of a bunch of clicking & typing

Reply:
ok. will do